src/uct/ib/base/ib_md.c

```diff
@@ -27,6 +27,16 @@
                                   IBV_ACCESS_REMOTE_ATOMIC)
 #define UCT_IB_MD_RCACHE_DEFAULT_ALIGN 16
 
+typedef struct uct_ib_md_pci_info {
+    double   bw;       /* bandwidth */
+    unsigned payload;  /* payload used to data transfer */
```

Comment:
could we make some fields uint16?

Reply:
actually all fields (except bw) could be uint16. fixed